statistics/handle/handle.go

```diff
@@ -562,14 +567,14 @@ func (h *Handle) UpdateStatsHealthyMetrics() {
 // Update reads stats meta from store and updates the stats map.
 func (h *Handle) Update(is infoschema.InfoSchema, opts ...cache.TableStatsOpt) error {
 	oldCache := h.statsCache.Load()
-	lastVersion := oldCache.Version()
+	lastVersion := h.latestUpdateTableVersion.Load()
```

Comment:
Is it better to write it like this?
```
h.statsCache.LatestUpdateTableVersion()
```

Reply:
No, `statsCache` should be unaware of this variable. This variable is only used when the StatsHandler reading stats data from the storage, so it's better to let the StatsHandler maintain this variable itself.